lib/registration.pm

```diff
@@ -120,11 +123,11 @@ sub fill_in_registration_data {
                 }
             }
             send_key 'alt-n';    # next
-            while (check_screen('import-untrusted-gpg-key', 20)) {
+            while (check_screen('import-untrusted-gpg-key')) {
                 record_soft_failure 'untrusted gpg key';
                 send_key 'alt-t', 2;
             }
-            sleep 20;            # scc registration need some time
+            sleep 30;            # scc registration need some time
```

Comment:
Not really. I am currently convinced that we don't need any explicit `sleep` or `check_screen` ever. We should be able to write everything e.g. like

```perl
assert_screen([qw/scc-finishes-correctly scc-comes-up-with-surprises/]);
if (match_has_tag('scc-comes-up-with-surprises')) {
    # do something special
    assert_screen 'scc-finishes-correctly';
}
```